app/src/test/java/com/example/vybe/VibeTest.java

```diff
@@ -1,135 +1,159 @@
-        assertEquals(R.drawable.ic_scared, scared.getEmoticon());
-
-        scared = Vibe.ofEmoticon(R.drawable.ic_scared);
-        assertEquals("Scared", scared.getName());
-        assertEquals(R.color.Blue, scared.getColor());
-        assertEquals(R.drawable.ic_scared, scared.getEmoticon());
-
-    }
-
-    @Test
-    public void GetHappyVibe() {
-        Vibe happy = Vibe.HAPPY;
-        assertEquals("Happy", happy.getName());
-        assertEquals(R.color.Yellow, happy.getColor());
-        assertEquals(R.drawable.ic_happy, happy.getEmoticon());
-
-        happy = Vibe.ofName("Happy");
-        assertEquals("Happy", happy.getName());
-        assertEquals(R.color.Yellow, happy.getColor());
-        assertEquals(R.drawable.ic_happy, happy.getEmoticon());
-
-        happy = Vibe.ofEmoticon(R.drawable.ic_happy);
-        assertEquals("Happy", happy.getName());
-        assertEquals(R.color.Yellow, happy.getColor());
-        assertEquals(R.drawable.ic_happy, happy.getEmoticon());
-
-    }
-
-    @Test
-    public void GetSadVibe() {
-        Vibe sad = Vibe.SAD;
-        assertEquals("Sad", sad.getName());
-        assertEquals(R.color.Gray, sad.getColor());
-        assertEquals(R.drawable.ic_sad, sad.getEmoticon());
-
-        sad = Vibe.ofName("SAD");
-        assertEquals("Sad", sad.getName());
-        assertEquals(R.color.Gray, sad.getColor());
-        assertEquals(R.drawable.ic_sad, sad.getEmoticon());
-
-        sad = Vibe.ofEmoticon(R.drawable.ic_sad);
-        assertEquals("Sad", sad.getName());
-        assertEquals(R.color.Gray, sad.getColor());
-        assertEquals(R.drawable.ic_sad, sad.getEmoticon());
-
-    }
-
-    @Test
-    public void GetSurprisedVibe() {
-        Vibe surprised = Vibe.SURPRISED;
-        assertEquals("Surprised", surprised.getName());
-        assertEquals(R.color.Orange, surprised.getColor());
-        assertEquals(R.drawable.ic_surprised, surprised.getEmoticon());
-
-        surprised = Vibe.ofName("surprised");
-        assertEquals("Surprised", surprised.getName());
-        assertEquals(R.color.Orange, surprised.getColor());
-        assertEquals(R.drawable.ic_surprised, surprised.getEmoticon());
-
-        surprised = Vibe.ofEmoticon(R.drawable.ic_surprised);
-        assertEquals("Surprised", surprised.getName());
-        assertEquals(R.color.Orange, surprised.getColor());
-        assertEquals(R.drawable.ic_surprised, surprised.getEmoticon());
-
-    }
-
-    @Test(expected = IllegalArgumentException.class)
-    public void GetInValidVibe_ThrowException() {
-        Vibe.ofName("AntiVibe");
-    }
-}
+package com.example.vybe;
+
+import com.example.vybe.Models.Vibe;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+/**
+ * Tests the VibeFactory Class and Subclasses
+ *
+ * These tests ensure that the data of a Vibe is properly stored and returned from a Vibe Object
+ */
+public class VibeTest {
+
```

Comment:
does @TheBroMoe still need to update these tests? Or did you do it already @PranavB6

Reply:
I already did it